GeoLib/Station.cpp

```diff
@@ -24,18 +24,14 @@
 namespace GeoLib
 {
 Station::Station(double x, double y, double z, std::string name)
-    : Point(x, y, z),
-      _name(std::move(name)),
-      _type(Station::StationType::STATION),
-      _station_value(0.0),
-      _sensor_data(nullptr)
+    : Point(x, y, z), _name(std::move(name))
+
 {}
 
 Station::Station(Point* coords, std::string name)
     : Point(*coords),
       _name(std::move(name)),
-      _type(Station::StationType::STATION),
-      _station_value(0.0),
+
```

Comment:
Please remove the empty line.

Reply:
Also dropped the redundant `_sensor_data{nullptr}`.